src/mpc-hc/SubtitlesProvider.h

```diff
@@ -46,7 +46,7 @@ public:
         return std::make_shared<P>(pOwner);                                            \
     }                                                                                  \
 private:                                                                               \
-    virtual std::string Name() const override { return #P; }                           \
+    virtual std::string Name() const override { return N; }                           \

```

Comment:
Don't modify Name(). Leave it as `#P`. Otherwise it breaks stored logins.
Add DisplayName() function as `N`.
In `CPPageSubMisc::OnInitDialog()` replace Name() with DisplayName(). Adjust width of first column to make new names fit.